src/browser/modules/Sidebar/Sidebar.tsx

```diff
@@ -152,18 +155,19 @@ const Sidebar = ({
             }
           }
         ]
-      : []),
+      : [])
+  ]
+
+  const bottomNavItemsList: NavItem[] = [
     {
       name: 'Documents',
       title: 'Help &amp; Resources',
       icon: function docsIcon(isOpen: boolean): ReactElement {
         return <DocumentsIcon isOpen={isOpen} title="Help &amp; Resources" />
       },
-      content: DocumentsDrawer
-    }
-  ]
-
-  const bottomNavItemsList = [
+      content: DocumentsDrawer,
+      enableCannyBadge: true
```

Comment:
In what situation would we change this to false? A future setting?

Reply:
It's never false, it's true or undefined. We could remove this flag and when building the jsx for the button instead check the name for documents instead of presence of this flag

Reply:
Ah had another look and saw why it's needed